Dockerfile

```diff
@@ -34,6 +34,14 @@ ENV ENABLE_WEBHOOKS=${ENABLE_WEBHOOKS}
 # Refer to https://catalog.redhat.com/software/containers/ubi8/ubi-minimal/5c359a62bed8bd75a2c3fba8 for more details
 FROM registry.access.redhat.com/ubi8/ubi-minimal:8.9-1029
 COPY --from=builder /opt/app-root/src/manager /
+
+# It is mandatory to set these labels
+LABEL description="RHTAP Integratioin Service"
```

Comment:
Shall we use upstream name `Appstudio` instead?

Reply:
Hm, good question. The other services are using `RHTAP`, e.g. the [build-service](https://github.com/redhat-appstudio/build-service/blob/2cc8fdd36e01c703667b69d64331bdaab33edc71/Dockerfile#L29C23-L29C23) and [release-service](https://github.com/redhat-appstudio/release-service/blob/9aea1dba75c09271dc60770a6fe434232cd654b6/Dockerfile#L35), but we could check if this is the way to go.